runtime/shibuya/src/lib.rs

```diff
@@ -440,6 +440,7 @@ parameter_types! {
     pub const MaxCandidates: u32 = 200;
     pub const MinCandidates: u32 = 5;
     pub const MaxInvulnerables: u32 = 20;
+    pub const SlashRatio: Perbill = Perbill::from_percent(10);
```

Comment:
Just a question, on which amount will be taken the slash? Is it address total amount or bonded funds?

Reply:
This is 10% of staked amount